numpy/core/tests/test_multiarray.py

```diff
@@ -3332,7 +3332,43 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kw):
         with assert_raises(NotImplementedError):
             a ** 2
 
-
+    def test_fast_scalar_pow(self):
+        # test for fast exponentiation on arrays
+        class SomeClass(object):
```

Comment:
A comment explaining that `__mul__` and `__div__` are deliberately missing, and that this tests that `**` is _not_ taking a fast path via them.

Better would be to add`
```
def __mul__(self, other):
    raise AssertionError('__mul__ should not be called')
```